lib/shipshape/actions/aws/parameter_store.rb

```diff
@@ -19,9 +19,10 @@ def get_parameters(path)
           params.map { |param_obj| param_obj.to_h.select { |key,| %i[name value].include?(key) } }
         end
 
-        def put_parameters(*params)
+        def put_parameters(prefix, *params)
           params.each { |param|
             ssm_client.put_parameter(put_opts.merge(param))
+            ssm_client.add_tags_to_resource(parameter_tags(param[:name], prefix))
```

Comment:
Apparently you can't add them as a parameter of `#put_parameter` if you also use `overwrite: true`